src/analysis/normed/group/seminorm.lean

```diff
@@ -64,8 +74,15 @@ and such that `f x⁻¹ = f x` and `f x = 0 → x = 1` for all `x`. -/
 structure group_norm (G : Type*) [group G] extends group_seminorm G :=
 (eq_one_of_map_eq_zero' : ∀ x, to_fun x = 0 → x = 1)
 
-attribute [nolint doc_blame] add_group_seminorm.to_zero_hom  add_group_norm.to_add_group_seminorm
-  group_norm.to_group_seminorm
+/-- A nonarchimedean norm on an additive group `G` is a function `f : G → ℝ` that preserves zero, is
+nonarchimedean and such that `f (-x) = f x` and `f x = 0 → x = 0` for all `x`. -/
+@[protect_proj]
+structure nonarch_add_group_norm (G : Type*) [add_group G] extends nonarch_add_group_seminorm G :=
```

Comment:
Why not
```suggestion
structure nonarch_add_group_norm (G : Type*) [add_group G] extends nonarch_add_group_seminorm G, add_group_norm G :=
```
```suggestion
structure nonarch_add_group_norm (G : Type*) [add_group G] extends nonarch_add_group_seminorm G :=
```

Reply:
I didn't define it this way to avoid having the superfluous `add_le'` field, but I don't have strong feelings about this, I could change the definition if there is greater support for this version.

Reply:
I added this comment.